libs/perun/components/src/lib/services-status-list/services-status-list.component.html

```diff
@@ -0,0 +1,101 @@
+<div class="card mt-3" [class.hide-table]="exporting"
+     [hidden]="dataSource.filteredData.length === 0 || servicesStatus.length === 0">
+  <div class="card-body">
+    <perun-web-apps-table-options (exportEnd)="exporting = false" (exportStart)="exporting = true" [exporter]="exporter"
+                                  class="ml-auto"></perun-web-apps-table-options>
+    <table mat-table matTableExporter [dataSource]="dataSource" #exporter="matTableExporter" matSort matSortActive="id"
+           matSortDirection="asc" matSortDisableClear
+           class="w-100">
+
+      <ng-container matColumnDef="select">
+        <th mat-header-cell *matHeaderCellDef>
+          <mat-checkbox color="primary"
+                        (change)="$event ? masterToggle() : null; this.selectionChange.emit()"
+                        [checked]="selection.hasValue() && isAllSelected()"
+                        [indeterminate]="selection.hasValue() && !isAllSelected()"
+                        [aria-label]="checkboxLabel()">
+          </mat-checkbox>
+        </th>
+        <td mat-cell *matCellDef="let serviceState" class="static-column-size">
+          <mat-checkbox color="primary"
+                        (click)="$event.stopPropagation()"
+                        (change)="$event ? selection.toggle(serviceState) : null; this.selectionChange.emit()"
+                        [checked]="selection.isSelected(serviceState)"
+                        [aria-label]="checkboxLabel(serviceState)">
+          </mat-checkbox>
+        </td>
+      </ng-container>
+
+      <ng-container matColumnDef="id">
+        <th mat-header-cell *matHeaderCellDef
+            mat-sort-header>{{'SHARED_LIB.PERUN.COMPONENTS.SERVICES_STATUS_LIST.TASK_ID' | translate}}</th>
+        <td mat-cell class="static-column-size" *matCellDef="let serviceState">
+          <div *ngIf="serviceState.task">{{serviceState.task.id}}</div>
+          <div *ngIf="!serviceState.task">0</div>
+        </td>
+      </ng-container>
```

Comment:
mat-sort-header is not working in columns id, serviceName and started and ended. Just remove it or make it work.

Reply:
sorting now works on every column